nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -665,6 +668,9 @@ public Description matchTypeCast(TypeCastTree tree, VisitorState state) {
 
   @Override
   public Description matchParameterizedType(ParameterizedTypeTree tree, VisitorState state) {
+    if (!withinAnnotatedCode(state)) {
```

Comment:
Test for this one? I only see the test for type cast within unannotated code.

Reply:
4737566